src/include/gpopt/translate/CQueryMutators.h

```diff
@@ -145,6 +145,86 @@ class CQueryMutators
 
 	} CContextTLWalker;
 
+	// context for walker to collect unique TargetlistEntry references in groupClause
```

Comment:
`TargetlistEntry` is not a proper term. TargetEntry is more correct, check other comments for that

Reply:
done